### ISSUE TITLE
Inconsistency between documented argument name and client's kwargs

### ISSUE THREAD

**Issue by surlyengineer** [GetStream/stream-chat-python#181]
In the create portion of the docs for python [here](https://getstream.io/chat/docs/python/creating_channels/) , the creation example is given as:

```
channel = client.channel("messaging", "travel", custom_data=dict(members=["thierry"]))
channel.create("myuserid")
```

As written that'll throw a `TypeError: StreamChat.channel() got an unexpected keyword argument 'custom_data'`

This tracks with the definition of the channel interface [here](https://github.com/GetStream/stream-chat-python/blob/35da0fa759e88067c687a28d1ddf22b8c4826076/stream_chat/base/channel.py#L9) but conflicts with the client definition [here](https://github.com/GetStream/stream-chat-python/blob/master/stream_chat/base/client.py#L621)

I _believe_ this should be as easy as updating the docs to say: 

```
channel = client.channel("messaging", "travel", data=dict(members=["thierry"]))
channel.create("myuserid")
```

Or restoring the `kwarg` name to `custom_data` in the client definition.

**Comment by szuperaz** [comment]
Yes, this is the correct code: `channel = client.channel("messaging", "travel", data=dict(members=["thierry"]))`. A fix is on its way to our docs. Thanks for reporting this issue.

**Comment by szuperaz** [comment]
The documentation has been fixed: https://getstream.io/chat/docs/python/creating_channels/#2-creating-a-channel-for-a-list-of-members---distinct-channels
